docs/2.10.0/docs/upgrade/smart-contract-upgrades.rst

```diff
@@ -98,6 +98,16 @@ error-prone migrations, without downtime, without requiring any
 additional network traffic, and without any extensive communication with
 downstream users.
 
+With SCU, different versions of the _same_ package are now grouped under a common package-name.
+This allows template/interface IDs to now be unambiguously referenced using either:
+
+-  the package ID
+
+-  or by using the package-name (which is resolved by a Canton participant to a unambiguous package ID).
```

Comment:
Since we want to promote the best practice of providing package preferences in ledger API request, and also because it makes it sound less magical, I would rather communicate that a package is uniquely determined by a package ID or by a package name + a package preference (which is a set of preferred package IDs in which no two package IDs have the same name, i.e. an implicit mapping from package names to package IDs). This is an overview so this formulation is way too verbose. The point I'd like to get across is that the user has control over how the participant resolves names and more importantly, we recommend using that control.

Reply:
There is a latter section of the document (Dynamic Package Resolution in Ledger API Queries) that better describes this process. As a result I've removed this text from the overview.